.github/workflows/merge.yaml

```
@@ -105,20 +105,14 @@ jobs:
         curl -H 'Content-Length:' -H "Authorization: $ZOTERO_UPDATE_TOKEN" -F 'payload={"type":"push","branch":"${{ steps.release.outputs.branch }}","status":0,"commit":"'$GITHUB_SHA'"}' https://styles-update.zotero.org:8826/
         curl -H 'Content-Length:' -H "Authorization: $ZOTERO_UPDATE_TOKEN" -F 'payload={"type":"push","branch":"${{ steps.release.outputs.branch }}","status":0,"commit":"'$GITHUB_SHA'"}' https://styles-update.zotero.org:8827/
 
-    - name: copy workflows to locales -- clone locales
-      if: github.repository == 'citation-style-language/styles' && steps.update.outputs.workflows == 'true'
-      uses: actions/checkout@v2
-      with:
-        repository: 'citation-style-language/locales'
-        path: 'locales'
-    - name: copy workflows to locales -- copy workflows
-      if: github.repository == 'citation-style-language/styles' && steps.update.outputs.workflows == 'true'
-      run: |
-        mkdir -p locales/.github/workflows/
-        cp .github/workflows/*.yaml locales/.github/workflows/
-    - name: copy workflows to locales -- commit workflows
-      uses: stefanzweifel/git-auto-commit-action@v4
+    - name: Copy workflows to locales repository
       if: github.repository == 'citation-style-language/styles' && steps.update.outputs.workflows == 'true'
+      uses: drud/action-cross-commit@master
       with:
-        repository: 'locales'
-        commit_message: copied ${{ steps.update.outputs.workflows_files }} to locales
+        source-folder: .github/workflows
+        destination-repository: https://csl-bot:${{ secrets.CSLBOT }}@github.com/citation-style-language/locales
```

I think given the name of the secret, this needs to be

```suggestion
        destination-repository: https://csl-bot:${{ secrets.CSLBOT_TOKEN }}@github.com/citation-style-language/locales
```